CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Polish translation ([#690](https://github.com/GIScience/openrouteservice/issues/690))
 ### Fixed
+- Fixed fallback to dynamic routing methods if bearings parameter set ([#702](https://github.com/GIScience/openrouteservice/issues/702))
```

Comment:
Shouldn't this be rather part of another PR?

Reply:
it is, I branched off the wrong place...